internal/app/utils.go

```diff
@@ -374,22 +374,33 @@ func deleteFile(path string) {
 // and the webhook URL as well as a flag specifying if this is a failure message or not
 // It returns true if the sending of the message is successful, otherwise returns false
 func notifySlack(content string, url string, failure bool, executing bool) bool {
-	log.Info("Posting notifications to slack ... ")
+	log.Info("Posting notifications to Slack ... ")
 
 	color := "#36a64f" // green
 	if failure {
 		color = "#FF0000" // red
 	}
 
+	var contentBold string
 	var pretext string
+
 	if content == "" {
-		pretext = "No actions to perform!"
+		pretext = "*No actions to perform!*"
+		contentBold = content
```

Comment:
Nti: At this point both variables are empty strings,  this line seems redundant.

Reply:
I went ahead and removed it as suggested.